app/navigation/Navigator.scala

```diff
@@ -138,10 +138,34 @@ class Navigator @Inject()() {
   }
 
   private def navigationForStartDate(userAnswers: UserAnswers, mode: Mode): Call = {
-    if(userAnswers.get(page = StartDatePage).isDefined && mode == NormalMode) {
+    if (isLarge(userAnswers)) {
+      largeProducerToPackagingSiteOrWarehouse(userAnswers, mode)
+    } else {
+      notALargeProducerToPackagingSiteOrWarehouse(userAnswers, mode)
+    }
+  }
+
+  private def isSmall(userAnswers: UserAnswers): Boolean = UserTypeCheck.isSmall(userAnswers)
+  private def isLarge(userAnswers: UserAnswers): Boolean = UserTypeCheck.isLarge(userAnswers)
+  private def notAProducer(userAnswers: UserAnswers): Boolean = UserTypeCheck.notAProducer(userAnswers)
+  private def copackerAll(userAnswers: UserAnswers): Boolean = UserTypeCheck.copackerAll(userAnswers)
+  private def copackee(userAnswers: UserAnswers): Boolean = UserTypeCheck.copackee(userAnswers)
+  private def importer(userAnswers: UserAnswers): Boolean = UserTypeCheck.importer(userAnswers)
+  private def operatesPackagingSite(userAnswers: UserAnswers): Boolean = UserTypeCheck.operatesPackagingSite(userAnswers)
```

Comment:
are these private def's needed? they they just reference user typecheck

Reply:
It shortens the code needed within the method and will be used by the other tickets that have criteria for what page to show.